### ISSUE TITLE
remarks in specification of `<att>` have an ambiguous reference

### ISSUE THREAD

**Issue by GVogeler** [TEIC/TEI#1506]
http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.html states:

> A namespace prefix may be used in order to specify the scheme as an alternative to specifying it via the scheme attribute: it takes precedence

Clarify wether the word `it` in `it takes precedence` refers to the namespace prefix or to the `@scheme`.

**Comment by hcayless** [comment]
I think this has to mean an "attached" namespace prefix must take precedence over the `@scheme` attribute. Though it doesn't necessarily override it. `@scheme` doesn't actually specify a namespace, though it probably implies one. E.g. scheme="TEI" implies "http://www.tei-c.org/ns/1.0", but the attribute could be `@xml:id`, so both together would be fine. And of course, the prefix is technically meaningless, unless it's 'xml:', or, I suppose, if it's been defined in a prefixDef. I think I'm in favor of:

> A namespace prefix may be used on the attribute name, and may imply a scheme if one isn't specified in `@scheme`.

**Comment by ebeshero** [comment]
F2F subgroup (via @tuurma) suggests to reword the note thus:
````
As an alternative to using scheme attribute a namespace prefix may be used. Where both @scheme and a prefix are used, the prefix takes precedence
````

**Comment by emylonas** [comment]
closed with commit cb941ce
